sdk/ml/azure-ai-ml/azure/ai/ml/operations/_job_operations.py

```diff
@@ -1124,6 +1127,18 @@ def _resolve_finetuning_job_inputs(self, job: FineTuningJob) -> None:
             if job.validation_data is not None:
                 self._resolve_job_input(job.validation_data, job._base_path)
 
+    def _resolve_distillation_job_inputs(self, job: DistillationJob) -> None:
+        """This method resolves the inputs for Distillation jobs.
+
+        :param job: the job resource entity
+        :type job: DistillationJob
```

Comment:
we are using different version while creating the job, look at function "_create_or_update_with_different_version_api", you may need to add a if condition for new job type to ensure it usages correct api version.

Reply:
When we create the DistillationJob we end up creating a Finetuning REST object. FT added an overloaded API to allow for distillation. So in "_create_or_update_with_different_version_api" no if statement is needed as the rest object passed in should be a Finetune REST Job. Once everything if finalized in MFE, we will probably need to add the if statement as distillation will then have its own REST object and will stop using the overloaded FT API